tests/integration.rs

```diff
@@ -48,6 +48,32 @@ fn netflix() {
     );
 }
 
+#[test]
+fn sanofi() {
```

Comment:
How did you find a domain using this?

Reply:
This is extracted from the rustls bug pcap.